students/eowyn/session04/trigrams.py

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env python
+
+
+import random
+import string
+
+# pg 125 of think python has line processing example
+PUNCTUATION = ['.', '!', '?']
+
+# TO DO: figure out how to properly capitalize I, proper nouns
+# TO DO: Fix trigrams that end in words like "and", prepositional phrases, articles
+
+
+def convert_text(filename):
+    ''' 
+    Convert the full text of a local filename (str) to 
+    return the corresponding dictionary of trigrams
+    '''
+    words = []
+    for lines in open(filename):
```

Comment:
Good to get in the habit of using "with" with fileopening:

```
with open(filename) as infile:
    for line in file:
        ...
```

Also, better to use the singular: "line" than "lines" for what will be a single line.